add known issue for dr secondary lease count quota invalidation (#12288)
hashicorp/vault: website/content/partials/lease-count-quota-upgrade.mdx

### Lease Count Quota Invalidations on DR Secondaries Fixed

Lease count quota invalidation causes DR Secondaries to panic and experience

> a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It has been

Reviewer: ```suggestion
a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It is
```

Reviewer: It won't quite be released when this PR merges, so we can leave the tense ambiguous this way.
